code/modules/jobs/job_types/corporate_affairs_agent.dm

```diff
@@ -0,0 +1,63 @@
+/datum/job/caa
+	title = "Corporate Affairs Agent"
+	flag = CAA
+	department_head = list("Captain")
+	department_flag = CIVILIAN
+	faction = "Station"
+	total_positions = 2
+	spawn_positions = 2
+	supervisors = "Central Command"
+	selection_color = "#ddddff"
+	chat_color = "#50C878"
+	exp_requirements = 840
+	exp_type = EXP_TYPE_CREW
+
+	outfit = /datum/outfit/job/caa
+
+	access = list(ACCESS_CAA, ACCESS_COURT, ACCESS_SEC_DOORS, ACCESS_SEC_RECORDS,
+				ACCESS_MEDICAL, ACCESS_CONSTRUCTION, ACCESS_RESEARCH, ACCESS_CARGO,
+				ACCESS_HEADS)
+	minimal_access = list(ACCESS_CAA, ACCESS_COURT, ACCESS_SEC_DOORS, ACCESS_SEC_RECORDS,
+				ACCESS_MEDICAL, ACCESS_CONSTRUCTION, ACCESS_RESEARCH, ACCESS_CARGO,
+				ACCESS_HEADS)
+	paycheck = PAYCHECK_MEDIUM
+	paycheck_department = ACCOUNT_CIV
+	mind_traits = list(TRAIT_LAW_ENFORCEMENT_METABOLISM)
```

Comment:
Since they aren't security/cops, them loving donuts doesn't make as much sense.

Reply:
you are, in fact, correct. removed.